### ISSUE TITLE
Date modified gets lost when File is copied on RDS Host

### ISSUE THREAD

**Issue by Alexander-Philipp** [FreeRDP/FreeRDP#3380]
Hi,
we are using freerdp 2.0.0.4 and whenever a File is copied on a Windows 2012 R2 RDS Host the modified date gets lost (not visible anymore on Windows Explorer). File Properties Shows 1.1.1970 01:00:00 for modified date.
Users a connecting from CentOS 7 through a Windows 2012 RDS Gateway.

Is this bug already known?

Thanks in advance!
Alexander

**Comment by bmiklautz** [comment]
@Alexander-Philipp I'm not aware of version 2.0.0.4. Do you have a git tag or similar?

**Comment by Alexander-Philipp** [comment]
version 2.0.0-dev (git n/a)

Sry for the mix-up

Von: Bernhard Miklautz [mailto:notifications@github.com]
Gesendet: Donnerstag, 2. Juni 2016 09:45
An: FreeRDP/FreeRDP <FreeRDP@noreply.github.com>
Cc: Alexander Philipp <alexander.philipp@ssw-trading.com>; Mention <mention@noreply.github.com>
Betreff: Re: [FreeRDP/FreeRDP] Date modified gets lost when File is copied on RDS Host (#3380)


@Alexander-Philipp<https://github.com/Alexander-Philipp> I'm not aware of version 2.0.0.4. Do you have a git tag or similar?

**Comment by flega68** [comment]
Hi,

Same Problem here. All tough my date gets set to xx.07.2011.
xfreerdp version 2.0.0-dev (git 42d2b69)

**Comment by mydigia** [comment]
Hello,
We are having the same problem through our integration, can this issue be looked at please?

**Comment by akallabeth** [comment]
@mydigia Did you test with current master? This issue has already been fixed.

**Comment by cnzhx** [comment]
Hi there,

Sorry if I am posting this at the wrong place. I am not sure whether it is the same problem as this issue.

I think I have a similar problem on current git version from openSUSE Tumbleweed. The version of freerdp is,
`cnzhx@ostp:~> xfreerdp --version
This is FreeRDP version 2.0.0-dev (git n/a)`

When I am in KDE Plasma 5 on Tumbleweed, I use FreeRDP to connect to the Windows 7 desktop in my office. I mount a folder to the Windows 7 remote as a temporary directory.

Then, I create a file in the folder in the Linux named `timestamp_check`. Here is the details:
```
cnzhx@ostp:/home/cnzhx/test> stat timestamp_check
  File: timestamp_check
  Size: 0               Blocks: 0          IO Block: 4096   regular empty file
Device: 35h/53d Inode: 31702259    Links: 1
Access: (0600/-rw-------)  Uid: ( 1000/   cnzhx)   Gid: (  100/   users)
Access: 2017-10-24 21:01:18.710093029 +0100
Modify: 2017-10-24 21:00:57.441632100 +0100
Change: 2017-10-24 21:00:57.441632100 +0100
 Birth: -
```
It shows the Modify time is `21:00:57` in Linux. But on the Windows 7 remote desktop, this file has a "Modified" datetime as `24 October 2017, 20:55:10`. The "Created" time on Windows 7 seems correctly reflecting the  time of (`21:00:57`).

**Comment by akallabeth** [comment]
@cnzhx Try our nightlies, looks like the suse packages did not incorporate this patch as of now

**Comment by cnzhx** [comment]
@akallabeth Thank you! I finally found out that my problem is the same as issue #4173  that possibly fixed by #4175 . I will report my test with the master there.
